<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWindow</class>
 <widget class="QMainWindow" name="CalibrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manual Calibration</string>
  </property>
  <widget class="QWidget" name="main_frame_layout">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="head_upper">
       <attribute name="title">
        <string>Head Upper</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="head_upper_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="head_upper_widget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>758</width>
             <height>500</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="head_upper_container"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="head_bottom">
       <attribute name="title">
        <string>Head Bottom</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="head_bottom_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="head_bottom_widget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>758</width>
             <height>500</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="head_bottom_container"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="head_global">
       <attribute name="title">
        <string>Head Global</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QScrollArea" name="head_global_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="head_global_widget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>758</width>
             <height>500</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="head_global_container">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="head_center_lb">
                  <property name="text">
                   <string>Center Head Rotation:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="reset_head_center_btn">
                  <property name="text">
                   <string>Reset</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="footer_layout">
      <item>
       <widget class="QPushButton" name="full_reset_btn">
        <property name="text">
         <string>Full Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="save_btn">
        <property name="text">
         <string>Apply and Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
